specificatie/domain.yaml

```diff
@@ -590,6 +593,21 @@ components:
                         Dit kan een aangeboden stuk of een kadasterstuk zijn.
           items:
             type: string
+    KadastraalOnroerendeZaakFiliatie:
+      type: "object"
+      description: |
+                    De eigenschappen van de kadastraalOnroerendeZaak die gerelateerd is.
+      properties:
+        identificatie:
+          description: |
+                        De identificaties van de Kadastraal Onroerende Za(a)k(en).
```

Comment:
Moet de naam van deze properties niet 'identificaties' zijn, gelet op de description, en moet het dan ook niet een array zijn?

Reply:
Indien de description correct is dan toch even corrigeren in:

`De identificatie(s) van de Kadastraal Onroerende Za(a)k(en).`

Reply:
Description aangepast.